### ISSUE TITLE
twisted.python.modules.PythonPath documents a non-existent "moduleLoader" and "sysPath" ivars

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3387]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3387|
|Type|defect|
|Created|2008-08-13 15:23:43Z|

The docs should be talking about `_moduleLoader` and `_sysPath`, I suspect.

<details><summary>Searchable metadata</summary>

```
trac-id__3387 3387
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1218641023000000 1218641023000000
changetime__1252359711000000 1252359711000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

r24671, which resolved [#3388](https://github.com/twisted/twisted/issues/3388), changes the docs for each of these.  The `_moduleLoader` fix seems to have been correct, but the `_sysPath` fix may not have been.  See [#4009](https://github.com/twisted/twisted/issues/4009).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
